src/networks.json

```diff
@@ -77,6 +77,16 @@
     "ws": ["wss://ws.dome.cloud"],
     "explorer": "https://exp.tch.in.th"
   },
+  "20": {
+    "key": "20",
+    "name": "Elastos Smart Chain",
+    "shortName": "ESC",
+    "chainId": 20,
+    "network": "mainnet",
+    "multicall": "0x20205D3b6008bea1411bd4CaEA2D923FE231B6E5",
+    "rpc": ["https://rpc.glidefinance.io", "https://escrpc.elaphant.app"],
```

Comment:
The second RPC works fine but first one is throwing the error:
```
 Error: could not detect network (event="noNetwork", code=NETWORK_ERROR, version=providers/5.4.5)
 ```

Reply:
Can you remove the first one?

Reply:
Sorry we just spun up that node and it might have been re-syncing when you checked. Should be stable now

Reply:
@racollette Looks like I am getting CROS error,

```JS
Access to fetch at 'https://rpc.glidefinance.io/' from origin 'https://snapshot-networks.on.fleek.co' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```
But works when I run it in my local nodejs env, it just doesn't work in browser env

Reply:
Argh. Apologies again. Should be CORS flagged now. If for some reason it doesn't work this time I'll remove it from the list, but would prefer to have a back up on there.